fizzbuzz/FizzBuzz.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+class FizzBuzz
+{
+    private $rules = [];
+
+    public function __construct()
+    {
+        $this->rules[] = new MultipleOfThree();
+        $this->rules[] = new MultipleOfFive();
+    }
+
+    public function check($number)
+    {
+        $output = '';
+        foreach ($this->rules as $rule) {
+            $output .= $rule->output($number);
+        }
+
+        if (strlen($output)) {
+            return $output . PHP_EOL;
```

Comment:
wouldn't include PHP_EOL in the domain layer, what if someone does not want to print the result but to put it in a database instead? Newlines are presentation logic